app/controllers/documents_controller.rb

```diff
@@ -99,6 +101,16 @@ def validate_step
 
 private
 
+  def audit(activity_type, audit_comment = nil, *activity_information)
+    Audit.create!(
+      planning_application_id: @planning_application.id,
+      user_id: current_user.id,
```

Comment:
Can we set these associations with their associations rather than the IDs? Like:
```
planning_application: @planning_application,
user: current_user
```